packages/nextjs/components/JourneyCard.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image";
+import TrackedLink from "~~/components/TrackedLink";
+
+/**
+ * Site header
+ */
+export const Card = ({
```

Comment:
Here we can name it as `JourneyCard` and update the import and usage in `index.tsx`, so it is more coherent with the other card naming (BuildCard and LearnMoreCard)

Reply:
Just added the fix but I'm getting this error, so I think I did something wrong 🤔
Changed the name in `index.tsx` and the `export const` in `JourneyCard`. Might be missing something

<img width="778" alt="Captura de pantalla 2024-09-30 a las 18 14 29" src="https://github.com/user-attachments/assets/f84aa942-6f16-4a1a-a9a9-6fe3823a3c11">

Reply:
In `index.tsx` did you update both the import and the <Card> elements?

Reply:
I did, but I forgot to save the JourneyCard.tsx file 😅 okay I got it now!